src/docs/product/integrations/integration-platform/ui-components/alert-rule-action.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Alert Rule Action
+sidebar_order: 2
+description: "Learn about the alert rule action UI component."
+---
+
+The alert rule action component gives you to access to parameters to define routing or configuration in alert rules for your service. When alert rules are triggered for a user, the configured service will receive [issue alert](/product/integrations/integration-platform/webhooks/#issue-alerts) and [metric alert](/product/integrations/integration-platform/webhooks/#metric-alerts) webhook events, with the specified settings for that service, to further route the alert.
+
+![Example of an alert rule UI component.](alert-rule-component-action-example.png)
+
+![Example of an alert rule settings modal.](alert-rule-component-modal-example.png)
+
+## Schema
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "type": "alert-rule-action",
+      "title": <String>,
+      "settings": {
+        "type": "alert-rule-settings",
+        "uri": <URI>,
+        "required_fields": <Array<FormField>>,
+        "optional_fields": <Array<FormField>>,
+        "description": <String>
+      }
+    }
+  ]
+}
+```
+
+## Attributes
+
+- `title` - (Required) The title shown in the UI component.
+- `uri` - (Required) Sentry will make a POST request to the URI when the User submits the form. If the services fails to process the request (status code >= 400), this component will bubble up the error to the User with the provided response text.
+- `required_fields` - (Required) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User is required to complete.
+- `optional_fields` - (Optional) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User may complete.
+- `description` - (Optional) Text that will be displayed above the form. Limited to 140 characters.
+
+## Example
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "type": "alert-rule-action",
+      "title": "Create a Rekall Inc task",
+      "settings": {
+        "type": "alert-rule-settings",
+        "uri": "/sentry/alert-rule",
+        "required_fields": [
+          {
+            "type": "select",
+            "label": "Project",
+            "name": "project",
+            "options": [["1", "cornflakes"]]
+          },
+          {
+            "type": "select",
+            "label": "Assignee",
+            "name": "assignee",
+            "url": "/sentry/alert-rule/options/users/"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+
+## Issue Alert Request Format
+
+When an issue alert fires, your service will need to read the settings from the alert payload. The `settings` are in `data.issue_alert.settings`. Check out the full [Issue Alert webhook documentation](/product/integrations/integration-platform/webhooks/#issue-alerts) for more information.
+
+### Payload
```

Comment:
I don't think we need this heading, they seem odd on the sidebar and don't have any text underneath.